src/execution/operator/persistent/buffered_csv_reader.cpp

```diff
@@ -1772,6 +1796,31 @@ void BufferedCSVReader::Flush(DataChunk &insert_chunk) {
 			}
 		}
 	}
+	if (conversion_error_ignored && options.ignore_errors) {
+		SelectionVector succesful_rows;
+		succesful_rows.Initialize(parse_chunk.size());
+		idx_t sel_size = 0;
+
+		for (idx_t row_idx = 0; row_idx < parse_chunk.size(); row_idx++) {
+			bool failed = false;
+			for (idx_t column_idx = 0; column_idx < sql_types.size(); column_idx++) {
+
+				auto &inserted_column = insert_chunk.data[column_idx];
+				auto &parsed_column = parse_chunk.data[column_idx];
+
+				bool was_already_null = FlatVector::IsNull(parsed_column, row_idx);
+				if (!was_already_null && FlatVector::IsNull(inserted_column, row_idx)) {
+					failed = true;
+					break;
+				}
+			}
+			if (!failed) {
+				succesful_rows.set_index(sel_size++, row_idx);
+			}
+		}
+		insert_chunk.Slice(succesful_rows, sel_size);
+		// insert_chunk.Verify();
```

Comment:
Should this be commented out?

Reply:
I wasn't sure if I should call Verify in this case, as the validity of the chunk shouldn't be disturbed by this operation.
And Verify is marked as a debug function

What would be the appropriate action, removing it or uncommenting it?

Reply:
Removing it is fine